cmd/libguestfs/BUILD.bazel

```diff
@@ -4,21 +4,39 @@ load(
 )
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 
-pkg_tar(
-    name = "appliance_layer",
-    srcs = ["@libguestfs-appliance//file"],
-    mode = "0444",
-    package_dir = "/usr/local/lib/guestfs",
-    visibility = ["//visibility:public"],
-)
-
 pkg_tar(
     name = "entrypoint",
     srcs = [":entrypoint.sh"],
     mode = "0775",
     package_dir = "/",
 )
 
+genrule(
+    name = "done-file",
+    outs = ["done"],
+    cmd = "touch $@",
```

Comment:
I'd like to avoid creating it together with the appliance. This is more about how kubevirt understands when the pod is ready for the tests. I'd like to remove it  after a reasonable amount of time.
How would you do with a local file? The `genrule` simply creates it at buildtime. It seems to me a nice way to do it instead of carry some extra files

Reply:
Fair enough, simply running `touch` should be pretty low overhead.